packages/react-core-notifications/src/components/index.ts

```diff
@@ -0,0 +1 @@
+export { RenderNothing } from './RenderNothing';
```

Comment:
Let's just import this from `@aws-amplify/ui-react-core` where it is used in in-app since `RenderNothing` has uses outside of in-app